### ISSUE TITLE
Love your App

### ISSUE THREAD

**Issue by rainbat** [ypwhs/NNPlayground#1]
Of course I can't read the labels in the app 😀 
But its a great tool on my iPhone to explain people Neural Networks anywhere ...

Love it and thank you for your work,
Domi from Switzerland

**Comment by ypwhs** [comment]
Thank you!!!
If you can do iOS programming, and are willing to help us make a English version, we will be very grateful to you!

**Comment by rainbat** [comment]
Yes why not - i have xcode and apple developer programm so i will have a look at it ! 

Cheers Dom

**Comment by ypwhs** [comment]
English version has merged: https://github.com/ypwhs/NNPlayground/pull/3
